pkg/main/main.go

```diff
@@ -28,6 +27,7 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	_ "sigs.k8s.io/controller-runtime/pkg/healthz"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
 
 	hrscheme "github.com/fluxcd/helm-operator/pkg/client/clientset/versioned/scheme"
```

Comment:
This needs changing to `github.com/fluxcd/helm-operator/pkg/apis/helm.fluxcd.io/v1`

Reply:
I think you meant line 32.  But yeah, agreed.